### PR TITLE
Design

### PR DESCRIPTION
**Pull request by Mhamad-Raad**
# Design
## In this section we have done the followings:
- Added the design of the Application.

### CODE REVIEW

File: src/Components/UI/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      </div>
+
+      <div className="row">
+        <button type="button" className="calcButton zero">0</button>
+        <button type="button" className="calcButton dot">.</button>
+        <button type="button" className="calcButton orange equal">=</button>
+      </div>
+    </div>
+  </div>
+);
```
Comment (ShahierNashaat) [inline src/Components/UI/Calculator.js]: - [ ] Kindly notice that the Calculator.js component should be class-based.

File: src/App.js [hunk 2]
```diff
@@ -1,9 +1,7 @@
 import './App.css';
+import React from 'react';
+import Calculator from './Components/UI/Calculator';
 
-function App() {
-  return (
-    <div className="App" />
-  );
-}
+const App = () => <Calculator />;
```
Comment (ShahierNashaat) [inline src/App.js]: - [ ] Kindly notice that the `App.js` component should be class-based.

File: src/Components/UI/Calculator.css [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+.row {
+  display: flex;
+}
+
+.calcButton {
+  color: black;
+  background-color: rgb(248, 248, 248);
+  border: 1px solid rgb(222, 222, 222);
+  font-size: 40px;
+  width: 400px;
```
Comment (ShahierNashaat) [inline src/Components/UI/Calculator.css]: - [ ] Kindly make the `width: 25%;` for the `calcButton` so the design looks good and the external space on the right is gone.

Capture

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @Mhamad-Raad, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 

- Correct use of GitFlow ✔️
- Descriptive commit messages ✔️
- Professional looking pull request ✔️
- Your readme looks professional ✔️
- Good overall design ✔️
- Your code is clean and well documented ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mateo951 — APPROVED** [review]
### Hi @Mhamad-Raad, 

Well done! Changes are applied correctly 🔥 Your project is complete!  There is nothing else to say other than... it's time to merge it 🐐 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**